lib/xdg.mli

```diff
@@ -6,10 +6,4 @@
 val config_dir : string
 (** The directory where the application should read/write config files. *)
 
-val data_dir : string
```

Comment:
I think we could leave `Xdg` out of this since it's kind of vendored. What do you think?

Reply:
Maybe the right solution here actually is to make it a standalone library in a `vendors/` dir at the root of the project!

Reply:
I will revert it and we can do that in another PR.